service/azureconfig/framework.go

```diff
@@ -61,58 +61,60 @@ func NewFramework(config FrameworkConfig) (*framework.Framework, error) {
 		return nil, microerror.Maskf(invalidConfigError, "config.TemplateVersion must not be empty")
 	}
 
-	var v1ResourceSet *framework.ResourceSet
+	var crdClient *k8scrdclient.CRDClient
```

Comment:
The diff is weird because I aligned the order of initialization to the other operators.